src/context/AppReducer.tsx

```diff
@@ -1,20 +1,19 @@
-import {IAppContext} from "./GlobalState";
+import {IAppContext, initialContext} from "./GlobalState";
 
-export type Action = {
-    type: ActionTypes,
-    newAddress: string
+type Action = {
+    type: string
+    payload?: any
 }
-
-export enum ActionTypes {
-    UpdateAddress
```

Comment:
Why did you remove the existing entry? We still need the 'update address' functionality.